GiftedMessenger.js

```diff
@@ -647,6 +669,7 @@ GiftedMessenger.defaultProps = {
 
 GiftedMessenger.propTypes = {
   autoFocus: React.PropTypes.bool,
+  backgroundImage: React.PropTypes.number,
```

Comment:
Opps, I misread it, thought it was the style `backgroundImage`. Anyway, should it be a string in case someone wants to pass an URL?